docs/FAQ.md

```diff
@@ -0,0 +1,10 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+## FAQ
```

Comment:
Let's add this to the product FAQ. 

It is here https://github.com/Microsoft/accessibility-insights-docs/blob/master/docs/web/reference/faq.md